webpack/src/Components/EmptyState/EmptyState.js

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { Button, EmptyState as PfEmptyState } from 'patternfly-react';
+import { useSelector, useDispatch } from 'react-redux';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { selectEmptyStateHeader } from './EmptyPageSelectors';
+import { AddEmptyStateHeader } from './EmptyStateActions';
+
+const EmptyState = ({ description }) => {
+  const dispatch = useDispatch();
+  const header = useSelector(selectEmptyStateHeader);
+
+  useEffect(() => {
+    dispatch(AddEmptyStateHeader(__('Foreman Plugin Template - React Page')));
```

Comment:
can you add an `index.js` file where it will trigger the action
while passing only the `header` prop into this component?